tests/charts/test_keda.py

```diff
@@ -52,6 +52,50 @@ def test_keda_enabled(self, executor, is_created):
         else:
             assert docs == []
 
+    @parameterized.expand(
+        [
+            ('CeleryExecutor'),
+            ('CeleryKubernetesExecutor'),
+        ]
+    )
+    def test_keda_advanced(self, executor):
+        """
+        Verify keda advanced config.
+        """
+        expected_advanced = {
+            "horizontalPodAutoscalerConfig": {
+                "behavior": {
+                    "scaleDown": {
+                        "stabilizationWindowSeconds": 300,
+                        "policies": [{"type": "Percent", "value": 100, "periodSeconds": 15}],
+                    }
+                }
+            }
+        }
+        docs = render_chart(
+            values={
+                "workers": {
+                    "keda": {
+                        "enabled": True,
+                        "advanced": {
+                            "horizontalPodAutoscalerConfig": {
+                                "behavior": {
+                                    "scaleDown": {
+                                        "stabilizationWindowSeconds": 300,
+                                        "policies": [{"type": "Percent", "value": 100, "periodSeconds": 15}],
+                                    }
+                                }
+                            }
+                        },
+                    },
+                    "persistence": {"enabled": False},
+                },
+                'executor': executor,
```

Comment:
```suggestion
                "executor": executor,
```

Nit: lets keep consistent quotes.

Reply:
Thanks, I fixed it.